talk/owt/sdk/base/customizedvideodecoderfactory.cc

```diff
@@ -24,7 +24,18 @@ CustomizedVideoDecoderFactory::~CustomizedVideoDecoderFactory() {}
 std::unique_ptr<webrtc::VideoDecoder>
 CustomizedVideoDecoderFactory::CreateVideoDecoder(
     const webrtc::SdpVideoFormat& format) {
-  return CustomizedVideoDecoderProxy::Create(external_decoder_->Copy());
+  if (absl::EqualsIgnoreCase(format.name, cricket::kVp8CodecName)) {
```

Comment:
Creating 4 identical calls with only 1 param different is not good.  Please add in owt/base/codecutils class an helper API to map SdpVideoFormat to VideoCodecType, and unify all decoder proxy instantiation to single call.